VideoApp/VideoApp/Stores/CrashReport/CrashReportStore.swift

```diff
@@ -14,7 +14,7 @@
 //  limitations under the License.
 //
 
-import FirebaseCrashlytics
+import Foundation
```

Comment:
Thanks for the PR @tihimsm! 

I think this change works for debug builds but not release builds because of line 35 below.

Reply:
Oh, I see.

Sorry, this PR is my misunderstanding.

Reply:
No problem!